pyiron/atomistics/master/elastic.py

```diff
@@ -0,0 +1,385 @@
+    strain_lst *= max_strain
+    m = np.linspace(-1, 1, int(2*polynomial_order+2*additional_points-1))
+    strain_lst = np.einsum('k,nij->nkij', m, strain_lst).reshape(-1, 9)
+    return np.unique(strain_lst, axis=0).reshape(-1, 3, 3)
+
+class ElasticJobGenerator(JobGenerator):
+    @property
+    def parameter_list(self):
+        parameter_lst = []
+        for ii, epsilon in enumerate(self._job.input['strain_matrices']):
+            basis = self._job.ref_job.structure.copy()
+            basis.apply_strain(np.array(epsilon))
+            parameter_lst.append([ii, basis])
+        return parameter_lst
+
+    def job_name(self, parameter):
+        return "{}_{}".format(self._job.job_name, parameter[0]).replace(".", "_")
+
+    @staticmethod
+    def modify_job(job, parameter):
+        job.structure = parameter[1]
+        return job
+
+class ElasticTensor(AtomisticParallelMaster):
+    """
+    Class to calculate the elastic tensor and isotropic elastic constants
+
+    Example:
+
+    >>> job = pr.create_job('SomeAtomisticJob', 'atomistic')
+    >>> job.structure = pr.create_structure('Fe', 'bcc', 2.83)
+    >>> elastic = job.create_job('ElasticTensor', 'elastic')
+    >>> elastic.run()
+
+    Input parameters:
+
+        min_num_measurements (int): Minimum number of measurements/simulations to be launched
+        min_num_points (int): Minimum number of data points to fit data (number of measurements
+            times number of symmetry operations)
+        strain_matrices (numpy.ndarray): Strain tensors to be applied on simulation boxes (optional)
+        rotations (numpy.ndarray): Rotation matrices for box symmetry
+        use_symmetry (bool): Whether or not exploit box symmetry (ignored if `rotations` already specified)
+        use_elements (bool): Whether or not respect chemical elements for box symmetry (ignored if `rotations`
+            already specified)
+
+    The default input parameters might not be chosen adequately. If you have a large
+    computation power, increase `min_num_measurements`. At the same time, make
+    sure to choose an orientation which maximizes the number symmetry operations.
+    Also if the child job does not support pressure, better increase the number
+    of measurements.
+    """
+    def __init__(self, project, job_name):
+        super().__init__(project, job_name)
+        self.__name__ = "ElasticTensor"
+        self.__version__ = "0.1.0"
+
+        self.input["min_num_measurements"] = (11, "minimum number of samples to be taken")
+        self.input["min_num_points"] = (105, "minimum number of data points"
+            + "(number of measurements will be min_num_points/len(rotations))")
+        self.input["max_strain"] = (
+            0.01,
+            "relative volume variation around volume defined by ref_ham",
+        )
+        self.input['polynomial_order'] = 2
+        self.input['additional_points'] = 0
+        self.input['strain_matrices'] = []
```

Comment:
Nit: Please add attributes in the same order as they appear in the docstring. This makes it easier to see when a docstring element is missing.

Reply:
I guess the order was the same but there were simply quite a few attributes/keys missing.